main.py

```diff
@@ -827,6 +893,71 @@ def _send_flow_mods(
                 self._send_napp_event(switch, flow, "pending")
         return flow_mods
 
+    def _send_flows_by_switch_from_request(
+        self,
+        request: Request,
+        command: str,
+        force: bool,
+        by_switch=False,
+    ) -> JSONResponse:
+        """Send flows in the respective switches from request."""
+        content_type_json_or_415(request)
+        content_dict = get_json_or_400(request, self.controller.loop)
+        if not isinstance(content_dict, dict):
+            raise HTTPException(400, detail=f"Invalid payload: {content_dict}")
+
+        if not any(content_dict):
+            result = "The request body doesn't have any flows"
+            raise HTTPException(400, detail=result)
+        
+        switches = []
+        switch_ids = []
+        for dpid in content_dict:
+            flows = content_dict[dpid].get("flows", [])
```

Comment:
If the payload has the wrong type, like a list it'd still crash, for example:

```
    flows = content_dict[dpid].get("flows", [])
            ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'get'
```

Reply:
Verifying if it a `dict` now